guides/micronaut-data-jdbc-repository/src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+micronaut:
+  application:
+    name: micronautguide
+---
+#tag::datasource[]
+datasources:
+  default:
+    url: jdbc:mysql://localhost:3306/db
+    driverClassName: com.mysql.cj.jdbc.Driver
+    username: root
+    password: ''
+    schema-generate: CREATE_DROP
```

Comment:
We should not tell the user to set `CREATE_DROP` for schema generation. But to use flyway instead.